nodes/nodes-pods-configuring.adoc

```diff
@@ -0,0 +1,46 @@
+// For example:
+// * file name: assembly_my-assembly-a.adoc
+// * ID: [id='assembly_my-assembly-a_{context}']
+// * Title: = My assembly A
+//
+// The ID is used as an anchor for linking to the module.
+// Avoid changing it after the module has been published
+// to ensure existing links are not broken.
+//
+// In order for  the assembly to be reusable in other assemblies in a guide,
+// include {context} in the ID: [id='a-collection-of-modules_{context}'].
+//
+// If the assembly covers a task, start the title with a verb in the gerund
+// form, such as Creating or Configuring.
+// The `context` attribute enables module reuse. Every module's ID
+// includes {context}, which ensures that the module has a unique ID even if
+// it is reused multiple times in a guide.
+:context: nodes-pods-configuring
+[id='nodes-pods-configuring_{context}']
```

Comment:
Please remove "_{context}" because we're not reusing assemblies. (True for all the assemblies)